src/main/resources/templates/placeOrder.html

```diff
@@ -17,121 +17,121 @@
-            <div class="form-group col-md-6">
-                <label for="itemNumber">Add Item Number</label>
-                <input type="text" class="form-control" id="itemNumber" placeholder="Item number">
+            <div class="form-row">
+                <div class="form-group col-md-6">
+                    <label for="itemNumber">Add Item Number</label>
+                    <input type="text" class="form-control" id="itemNumber" placeholder="Item number" th:field="*{itemId}">
+                </div>
+                <div class="form-group col-md-6">
+                    <label for="quantity">Enter Quantity</label>
+                    <input type="text" class="form-control" id="quantity" placeholder="Quantity" th:field="*{quantity}">
+                </div>
             </div>
-            <div class="form-group col-md-6">
-                <label for="quantity">Enter Quantity</label>
-                <input type="text" class="form-control" id="quantity" placeholder="Quantity">
+            <div class="form-row">
+                <div class="form-group col-md-6">
+                    <label for="inlineFormCustomSelectPref">Shipping Vendor</label>
+                    <select class="custom-select my-1 mr-sm-2" id="inlineFormCustomSelectPref" th:field="*{carrier}">
+                        <option value="Choose" selected>Choose...</option>
+                        <option value="1">USPS</option>
+                        <option value="2">FedEx</option>
+                        <option value="3">UPS</option>
+                    </select>
+                </div>
             </div>
-        </div>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="inlineFormCustomSelectPref">Shipping Vendor</label>
-                <select class="custom-select my-1 mr-sm-2" id="inlineFormCustomSelectPref">
-                    <option selected>Choose...</option>
-                    <option value="1">USPS</option>
-                    <option value="2">FedEx</option>
-                    <option value="3">UPS</option>
-                </select>
+            <div class="form-row">
+                <div class="form-group col-md-6">
+                    <label for="itemNumber">Additional Notes</label>
+                    <input type="text" class="form-control" id="deliveryRequests" placeholder="Delivery Requests" th:field="*{deliveryRequest}">
+                </div>
             </div>
-        </div>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="itemNumber">Additional Notes</label>
-                <input type="text" class="form-control" id="deliveryRequests" placeholder="Delivery Requests">
-            </div>
-        </div>
-        <button type="submit" class="btn btn-primary">Place Order</button>
-    </form>
-</div>
+            <button type="submit" class="btn btn-primary">Place Order</button>
+        </form>
+    </div>
+</main>
```

Comment:
Moved the exit </main> tag to after the main content of the page. <main> represents the main content of the page.
https://www.w3schools.com/tags/tag_main.asp
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main